### PR TITLE
Added information where to report docs issues

### PR DESCRIPTION
**Pull request by f1ames**
Since we decided that all docs related issues should be reported in `ckeditor-dev` repo, I have added some information about it.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+All issues regarding CKEditor 4 Documentation should be reported in `ckeditor-dev` repository - https://github.com/ckeditor/ckeditor-dev/issues/new/choose.
```
Comment (AnnaTomanek) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
All issues regarding CKEditor 4 Documentation should be reported in the `ckeditor-dev` repository - https://github.com/ckeditor/ckeditor-dev/issues/new/choose.
```

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+Any issues reported here still will be moved to `ckeditor-dev` repository.
```
Comment (AnnaTomanek) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
Any issues reported here will be moved to the `ckeditor-dev` repository.
```

File: README.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 This is the official developer documentation project for CKEditor. It uses the customized CKEditor [JSDuck clone](https://github.com/ckeditor/jsduck) for compilation and is available online at <http://docs.ckeditor.com>.
 
+**All issues regarding CKEditor 4 Documentation should be reported in [`ckeditor-dev` repository](https://github.com/ckeditor/ckeditor-dev/issues/new/choose).**
```
Comment (AnnaTomanek) [inline README.md]:
```suggestion
**All issues regarding CKEditor 4 Documentation should be reported in the [`ckeditor-dev` repository](https://github.com/ckeditor/ckeditor-dev/issues/new/choose).**
```

**Comment by f1ames** [comment]
@Comandeer could you check if everything is fine and merge the PR if it's ok?

**Pull request merged**